packages/rancher-monitoring/rancher-windows-exporter/package.yaml

```diff
@@ -1,3 +1,5 @@
-url: local
-version: 104.0.0
+url: https://github.com/prometheus-community/helm-charts.git
+subdirectory: charts/prometheus-windows-exporter
+commit: 3f7bb0e012e81a6497643d855df584ad5fe8b87a
+version: 104.0.1
```

Comment:
`104.0.0` hasn't been released yet so no need to bump version

Reply:
Yeah, I haven't focused too specifically on the version listed in this PR since I'm not 100% aware of any other work going on in monitoring which might land before this gets merged. Once this is taken out of draft I'll make sure this is correct 👍